src/token/token.service.ts

```diff
@@ -40,6 +42,11 @@ export class TokenService {
     return '18';
   }
   async getPrice() {
+    const key = this.redisName ? `${this.redisName}_marketCap` : 'marketCap';
```

Comment:
It's weird 'cause you set this from another function that also calls this method 🤯

Reply:
Right, I can reroute the controller call to the `marketDetails` and use this as internal